README.md

```diff
@@ -204,7 +204,7 @@ This tool can be used to script the backup of your databases. Move the backup an
 * `COUCH_URL` - the URL of the CouchDB/Cloudant server e.g. `http://127.0.0.1:5984`
 * `COUCH_DATABASE` - the name of the database to act upon e.g. `mydb` (default `test`)
 * `COUCH_PARALLELISM` - the number of HTTP requests to perform in parallel when restoring a backup e.g. `10` (Default `5`)
-* `COUCH_BUFFER_SIZE` - the number of documents fetched and restored at once e.g. `100` (default `500`)
+* `COUCH_BUFFER_SIZE` - the number of documents fetched and restored at once e.g. `100` (default `500`). At using CouchBackup with [Transaction Engine](https://www.ibm.com/cloud/blog/announcements/ibm-cloudant-on-transaction-engine) `COUCH_BUFFER_SIZE` must be less than `2000` to avoid bad request errors.
```

Comment:
Nit: what do you think about 
```suggestion
* `COUCH_BUFFER_SIZE` - the number of documents fetched and restored at once e.g. `100` (default `500`). When using CouchBackup with [Cloudant on Transaction Engine](https://www.ibm.com/cloud/blog/announcements/ibm-cloudant-on-transaction-engine) `COUCH_BUFFER_SIZE` must be less than `2000` to avoid bad request errors.
```

Reply:
Done in https://github.com/cloudant/couchbackup/pull/337/commits/cab12970cc46b962ccffc1062b916a62cb3980f2